app/projectwizard.cpp

```diff
@@ -17,6 +17,8 @@
 #include "qgsvectorfilewriter.h"
 #include "qgsdatetimefieldformatter.h"
 #include <qgsmarkersymbollayer.h>
+#include <qgssymbol.h>
+#include <qgsmarkersymbol.h>
```

Comment:
Please remove the `qgsmarkersymbol.h` include, it is due to a mismatch of QGIS versions.

Reply:
Android build is using previous QGIS version where this file does not exist. That is why CI does not pass

Reply:
Oh I see. Can you please tell me which version of QGIS is used in the CI so that I can add #ifdef to support the later version I use ?

Reply:
Let's not add ifdefs for now. If you could just remove this code (I understand that for local desktop you will need to add this), upcoming aab PR is going to sort this thing out :)